src/v2/api/controllers/RoleController.ts

```diff
@@ -33,6 +47,25 @@ export class RoleController {
   }
 
   @Access(PERMISSION.ROLES_CREATE)
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: RoleResponse,
+  })
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n 
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @ApiBadRequestResponse({
```

Comment:
the CreateRoleWithGrantsDTO uses CreateRoleDTO and CreateGrantDTO so add the following fields
![image](https://github.com/fictadvisor/fictadvisor-api/assets/63953579/0056bbdc-1e9d-45b7-9c6d-5f72d7b338ec)
![image](https://github.com/fictadvisor/fictadvisor-api/assets/63953579/fcf8bc2f-cd8b-40a6-a86e-c29961d0bae1)
like this
![image](https://github.com/fictadvisor/fictadvisor-api/assets/63953579/f780621a-e035-4cd0-8af1-f62347dc5d9d)